server/src/test/java/org/opensearch/cluster/ClusterChangedEventTests.java

```diff
@@ -107,19 +107,19 @@ public void testBasicProperties() {
     }
```

Comment:
`testLocalNodeIsMasterWithDeprecatedMasterRole()` -> `testLocalNodeIsMasterWithDeprecatedClusterManagerRole()` and the comment above
https://github.com/opensearch-project/OpenSearch/blob/737843fcc447c4e9167edc1e24a7ed59c043241c/server/src/test/java/org/opensearch/cluster/ClusterChangedEventTests.java#L318

Reply:
I will change the `Master` in `testLocalNodeIsMaster`, but the `MasterRole` is used intentionally. Thanks!

Reply:
Updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/a87cede9b88976e0a043e7ea8670d3de2f400c1e